perfschema/const.go

```diff
@@ -58,21 +58,91 @@ var PerfSchemaTables = []string{
 }
 
 // ColumnSetupActors contains the column name definitions for table setup_actors, same as MySQL.
```

Comment:
Table names, column names, maybe we will need it later when implementing instrumentation points, :)